### PR TITLE
export RocksDB statistics

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
Export RocksDB internal statistics.
@BusyJay @siddontang PTAL

### CODE REVIEW

File: src/rocksdb_options.rs [hunk 1]
```diff
@@ -750,6 +783,10 @@ mod tests {
         opts.enable_statistics();
         opts.set_stats_dump_period_sec(60);
         assert!(opts.get_statistics().is_some());
+        assert!(opts.get_statistics_histogram_string(DBStatisticsHistogramType::DbSeekMicros)
+            .is_some());
+        assert_eq!(opts.get_statistics_ticker_count(DBStatisticsTickerType::BlockCacheMiss),
```
Comment (siddontang) [inline src/rocksdb_options.rs]: not cover get_and_reset_statistics_ticker_count ?

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @hhkbp2

**Comment by hhkbp2** [comment]
LGTM

**Pull request merged**